en/docs/learn/api-controller/advanced-topics/configuring-environment-specific-parameters.md

```diff
@@ -12,16 +12,26 @@ environments:
       endpoints:
           production:
               url: <production_endpoint_url>
-              config:
-                  retryTimeOut: <no_of_retries_before_suspension>
-                  retryDelay: <retry_delay_in_ms>
-                  factor: <suspension_factor>
           sandbox:
               url: <sandbox_endpoint_url>
               config:
+                  factor: <suspension_factor>
+                  suspendMaxDuration: <maximum_suspend_time_duration>
+                  suspendDuration: <suspend_time_duration>
+                  suspendErrorCode: 
+                      - <suspend_error_code_1>
+                      - <suspend_error_code_2>
+                      - ......................
+                      - <suspend_error_code_n>
                   retryTimeOut: <no_of_retries_before_suspension>
                   retryDelay: <retry_delay_in_ms>
-                  factor: <suspension_factor>
+                  retryErroCode:
+                      - <retry_error_code_1>
+                      - <retry_error_code_2>
+                      - ....................
+                      - <retry_error_code_n>
+                  actionSelect: <action_discard_or_fault>
+                  actionDuration: <action_time_duration>
```

Comment:
Shall we add these configs in the config section under the production endpoint to make it more consistent with the example in (Lines between 81-94)? Or change the example to align with the template?